app/src/main/java/ch/sdp/vibester/activity/BuzzerScreenActivity.kt

```diff
@@ -95,6 +96,7 @@ class BuzzerScreenActivity : GameActivity() {
      */
     fun startRoundBuzzer(ctx: Context, gameManager: GameManager) {
         gameIsOn = true
+        toggleBtnVisibility(R.id.skip, true)
         findViewById<LinearLayout>(R.id.answer).visibility=View.INVISIBLE
         val title = gameManager.getCurrentSong().getTrackName()

```

Comment:
Could you rename this to something more indicative, like trackName or something? Title may be confusing to understand without reading the code in the future